### PR TITLE
[Docs] Add README_CN.md

### PR DESCRIPTION
**Pull request by caizhongang**

### CODE REVIEW

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    title={OpenMMLab Human Pose and Shape Estimation Toolbox and Benchmark},
+    author={MMHuman3D Contributors},
+    howpublished = {\url{https://github.com/open-mmlab/mmhuman3d}},
+    year={2021}
+}
+```
+
+## 参与贡献
+
+我们非常欢迎用户对于 MMPose 做出的任何贡献，可以参考 [CONTRIBUTION.md](.github/CONTRIBUTING.md) 文件了解更多细节。
```
Comment (ly015) [inline README_CN.md:110]: MMPose -> MMHuman3D

File: README_CN.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+请参考[快速入门](docs/getting_started.md)文档学习 MMDetection 的基本使用。
+
+## 许可
+
+该项目采用 [Apache 2.0 license](LICENSE) 开源协议。部分支持的算法可能采用了[额外协议](docs/additional_licenses.md)。
+
+## 引用
+
+如果您觉得 MMPose 对您的研究有所帮助，请考虑引用它：
```
Comment (ly015) [inline README_CN.md:97]: MMPose -> MMHuman3D

File: README_CN.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+```bibtex
+@misc{mmhuman3d,
+    title={OpenMMLab Human Pose and Shape Estimation Toolbox and Benchmark},
```
Comment (ly015) [inline README_CN.md:101]: Update title

File: README_CN.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+## 基准与模型库
+
+更多详情可见 [模型库](docs/model_zoo.md)。
+
+已支持的算法：
+
+<details open>
+<summary>(click to collapse)</summary>
+
+- [x] SMPLify (ECCV'2016)
```
Comment (ly015) [inline README_CN.md:48]: Consider adding paper/project link here?

File: README_CN.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+- [MMFlow](https://github.com/open-mmlab/mmflow): OpenMMLab 光流估计工具箱与测试基准
+- [MMFewShot](https://github.com/open-mmlab/mmfewshot): OpenMMLab 少样本学习工具箱与测试基准
+- [MMHuman3D](https://github.com/open-mmlab/mmhuman3d)：OpenMMLab 人体参数化模型工具箱与测试基准
+
+## 欢迎加入 OpenMMLab 社区
+
+扫描下方的二维码可关注 OpenMMLab 团队的 [知乎官方账号](https://www.zhihu.com/people/openmmlab)，加入 OpenMMLab 团队的 [官方交流 QQ 群](https://jq.qq.com/?_wv=1027&k=GJP18SjI)
+
+<div align="center">
+<img src="docs/imgs/zhihu_qrcode.jpg" height="400" />  <img src="docs/imgs/qq_group2_qrcode.jpg" height="400" />
```
Comment (ly015) [inline README_CN.md:139]: Consider adding MMHuman3D QQ group QR code here.

File: README_CN.md (hunk 1 above)
Comment (caizhongang) [reply README_CN.md]: Fixed

File: README_CN.md (hunk 2 above)
Comment (caizhongang) [reply README_CN.md]: Fixed.

**Comment by yl-1993** [comment]
Change mmcv version upper limit to 1.5 in [install.md](https://github.com/open-mmlab/mmhuman3d/blob/main/docs/install.md#requirements)

File: README_CN.md (hunk 4 above)
Comment (caizhongang) [reply README_CN.md:48]: Added!

**Comment by caizhongang** [comment]
@yl-1993 Fixed MMCV version upper limit, please check.

File: README_CN.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+
+### 主要特性
+
+- **使用模块化的框架复现流行的算法**
+
+  MMHuman3D 重新实现了流行的算法，帮助用户只需一行代码即可完成复现。模块化的框架适合快速验证：用户可以在不修改代码的情况下调试不同的超参数甚至神经网络结构。
+
+- **通过一个统一的数据规范 HumanData 支持多种数据集**
+
+  通过一个规范工具箱的帮助，我们将所有的支持的数据集都对齐到一个统一的数据格式 *HumanData* . 我们同时也提供预处理完成的数据文件。
```
Comment (yl-1993) [inline README_CN.md:33]: "通过一个规范工具箱的帮助" -> "通过一个规范工具箱" ?

**Review by yl-1993 — APPROVED** [review]

File: README_CN.md (hunk 5 above)
Comment (caizhongang) [reply README_CN.md]: Added!

File: README.md [hunk 7]
```diff
@@ -57,29 +59,29 @@ Supported datasets:
...
+- [x] [PoseTrack18](https://posetrack.net/users/download.php) (CVPR'2018)
+- [x] [SURREAL](https://www.di.ens.fr/willow/research/surreal/data/) (CVPR'2017)
+- [x] [UP3D](https://files.is.tuebingen.mpg.de/classner/up/) (CVPR'2017)
 
 </details>
 
 We will keep up with the latest progress of the community, and support more popular methods and frameworks.
 
-If you have any feature requests, please feel free to leave an issue.
+If you have any feature requests, please feel free to leave an comment in the [wishlist](https://github.com/open-mmlab/mmhuman3d/issues/3).
```
[the remark this answers is not in the document]
Comment (caizhongang) [reply README.md]: Fixed!

File: README_CN.md (hunk 6 above)
Comment (caizhongang) [reply README_CN.md]: Fixed!

File: README_CN.md [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+</details>
+
+我们将跟进学界的最新进展，并支持更多算法和框架。
+
+如果您对MMHuman3D有任何功能需求，请随时在[愿望清单](https://github.com/open-mmlab/mmhuman3d/issues/3)中留言。
+
+
+## 快速入门
+
+请参考[快速入门](docs/getting_started.md)文档学习 MMDetection 的基本使用。
```
[the remark this answers is not in the document]
Comment (caizhongang) [reply README_CN.md]: Fixed!

**Comment by OpenMMLab-Team** [comment]
Hi @caizhongang ！First of all, we want to express our gratitude for your significant PR in the MMHuman3D project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤

**Pull request merged**